modules/luci-mod-status/root/usr/libexec/syslog/syslog

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
rework this file using tabs instead of spaces.

Reply:
done and sorry, missed those spaces.